contacts.json

```diff
@@ -0,0 +1 @@
+[{"name":"Matt Warnock","address":"152 Leigh Hunt Drive","phone":"07947670780","email":"drflamerock@yahoo.co.uk","notes":"gets nervous when running his apps"}]
```

Comment:
No need to add the database to version control. You can add this to the `.gitignore` file.